.bootstraprc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I opted for Twitter Bootstrap for now. Never used it but figure it could give us some styles to work with while we're figuring things out.